q2_moshpit/diamond/tests/test_diamond.py

```diff
@@ -0,0 +1,28 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2022, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+
+from qiime2.plugin.testing import TestPluginBase
+from .._method import eggnog_diamond_search
+from q2_types_genomics.reference_db import DiamondDatabaseDirFmt
+from q2_types_genomics.per_sample_data import ContigSequencesDirFmt
+
+
+class TestDiamond(TestPluginBase):
```

Comment:
It would be good to add a test for the feature-table action as well.

I'm slightly suspicious of the feature-ids in the resulting table, since it seems like it will end up being the first letter of each ID as the ID.